FusionIIIT/applications/placement_cell/models.py

```diff
@@ -330,6 +330,8 @@ def __str__(self):
 
 
 class PlacementRecord(models.Model):
+    # id = models.OneToOneField(ExtraInfo, on_delete=models.CASCADE, primary_key=True)
+    # id = models.OneToOneField(ExtraInfo, on_delete=models.CASCADE, primary_key=True)

```

Comment:
```suggestion
    # id = models.OneToOneField(ExtraInfo, on_delete=models.CASCADE, primary_key=True)
    # id = models.OneToOneField(ExtraInfo, on_delete=models.CASCADE, primary_key=True)

```

remove the code comments if not needed.